extensions/wikia/AdEngine/js/video/player/porvata/porvataPlayerFactory.js

```diff
@@ -56,6 +61,9 @@ define('ext.wikia.adEngine.video.player.porvata.porvataPlayerFactory', ['wikia.l
 				ima.getAdsManager().resume();
 			},
 			setVolume: function (volume) {
+				if (videoAd) {
+					videoAd.muted = volume === 0;
+				}
 				return ima.getAdsManager().setVolume(volume);
```

Comment:
why setting volume to `0` here is not enough? and we need to do it also in l49 in `volumeControl.js`?